browser/src/map/Clipboard.js

```diff
@@ -86,17 +86,49 @@ L.Clipboard = L.Class.extend({
 	},
 
 	// Attempt to cleanup unwanted elements
-	stripStyle: function(domNode) {
-		var styles = domNode.querySelectorAll('style');
-		for (var i = 0; i < styles.length; i++) {
-			styles[i].parentNode.removeChild(styles[i]);
+	stripStyle: function(htmlString) {
+		let startIndex = 1;
+
+		while (startIndex !== -1) {
+			startIndex = htmlString.indexOf('<style ');
+			if (startIndex !== -1) {
+				let endIndex = htmlString.indexOf('</style>');
+				let subString = htmlString.substr(startIndex, endIndex - startIndex + '</style>'.length);
+				htmlString = htmlString.replace(subString, '');
+			}
+		}
+
+		// Remove also inline styles.
+
+		// For double quotes.
+		startIndex = 1;
+		while (startIndex !== -1) {
+			startIndex = htmlString.indexOf('style="');
+			if (startIndex !== -1) {
+				let endIndex = htmlString.indexOf('"', startIndex + 'style="'.length + 1);
+				let subString = htmlString.substr(startIndex, endIndex - startIndex + 1);
+				htmlString = htmlString.replace(subString, "");
+			}
 		}
+
+		// For single quotes.
+		startIndex = 1;
+		while (startIndex !== -1) {
```

Comment:
could be a helper function, it is the same as above code block only different `'` vs `"`